akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/javadsl/EventProducerPushDestination.scala

```diff
@@ -58,7 +58,7 @@ object EventProducerPushDestination {
     new EventProducerPushDestination(
       Optional.empty(),
       acceptedStreamId,
-      (_, _) => Transformation.empty,
```

Comment:
What does `emtpy` do? Does it eat the event?

Reply:
There will be a runtime exception telling you that you need to define some kind of transformation. `empty` is the starting point of the Transformation builder, then you register the actual transformations on that.

For these producer push events we have decided that it's not required to define a transformation.